packages/ui/components/dialog/Dialog.tsx

```diff
@@ -161,8 +217,9 @@ export function DialogFooter(props: { children: ReactNode; className?: string; s
       )}
       <div
         className={classNames(
-          "flex justify-end space-x-2 pb-4 pt-4 rtl:space-x-reverse",
-          !props.showDivider && "pb-8"
+          "flex justify-end space-x-2 rtl:space-x-reverse",
+          isMobile ? "pb-2 pt-2" : "pb-4 pt-4",
+          !props.showDivider && (isMobile ? "pb-6" : "pb-8")
```

Comment:
you could use breakpoints here instead of isMobile. https://tailwindcss.com/docs/responsive-design

Reply:
Also added this.